ddl_checker/README.md

```diff
@@ -0,0 +1,58 @@
+## DDLChecker
+
+DDLChecker is a tool for checking if DDL SQL can be successfully executed by TiDB.
+
+## How to use
+
+```
+Usage of checker:
+  --host string
+        MySQL host (default "127.0.0.1")
+  --port int
+        MySQL port (default 3306)
+  --user string
+        MySQL username (default "root")
+  --password string
+        MySQL password (default "")
+  --schema string
+        MySQL schema (default "")
+
+cd ddl_checker
+go build
+./ddl_checker --host [host] --port [port] --user [user] --password [password] --schema [schrma]
+```
+
+## Modes
+
+You can switch modes using the `SETMOD` command.  
+Auto mode: The program will automatically synchronize the dependent table structure from MYSQL and delete the conflict table.   
+Query mode: The program will ask you before synchronizing the dependent table structure from MYSQL.  
+Manual mode: This program does not perform anything other than executing the input SQL.  
+SETMOD Usage: SETMOD <MODCODE>; MODCODE = [0(Auto), 1(Query), 2(Manual)] 
```

Comment:
Could the tool be made to recognize the strings "auto", "query" and "manual" instead of 0, 1, 2?

BTW usually we use "prompt" or "ask". The term "query" is confusing esp. when we're talking about SQL.